redhat-cop/casl-ansible
# Ensure DNS configuration has wildcards set for infra nodes

#### What does this PR do?
Adds back in the wildcard for the DNS configuration

#### How should this be manually tested?
Run a provisioning following the instructions in the top level README. Check that the wildcard DNS records work afterwards (+ check the DNS config to validate that they are there)

#### Is there a relevant Issue open for this?
resolves #23

#### Who would you like to review this?
cc: @redhat-cop/casl @etsauer
